attestation-service/src/lib.rs

```diff
@@ -134,11 +134,17 @@ impl AttestationService {
             .evaluate(reference_data_map, tcb.clone(), None)
             .await?;
 
-        let attestation_results =
-            AttestationResults::new(tee, result, None, Some(policy_engine_output), Some(tcb));
```

Comment:
It seems like we are no longer propagating `policy_engine_output` to the KBS. We do report it in the case of an error, but we might want to think about cases where the policy does not have a binary output. It is possible that a client might have different types of claims that map onto different types of resources available from the KBS. It seems like having the output of the policy reported to the KBS would be very useful in this type of situation.

Reply:
Please refer to my [answer](https://github.com/confidential-containers/kbs/pull/139#issuecomment-1672505902) in your comment of https://github.com/confidential-containers/kbs/pull/139.

Reply:
@fitzthum Now I have modified the processing of the policy engine output to include the evaluation report in the Attestation Results Token :)